src/components/autocomplete/autocomplete.js

```diff
@@ -114,7 +124,16 @@ export default class Autocomplete {
       const client = v.client;
       if (client && v.type === AUTOCOMPLETE_TYPE.SEARCH) {
         client.nextPage();
-        this.reduxStore.dispatch(autocompleteSearch(client, v.jsonKey, keyword, true));
+        // TODO handle conversational search with autocomplete
```

Comment:
Comment can be removed.
As I understood, autocomplete is now handling conversational search

Reply:
removed